odl/solvers/util/callback.py

```diff
@@ -385,39 +401,63 @@ class CallbackPrintTiming(SolverCallback):
 
     """Callback for printing the time elapsed since the previous iteration."""
 
-    def __init__(self):
-        """Initialize a new instance."""
+    def __init__(self, fmt='Time elapsed = {:<5.03f} s', step=1):
```

Comment:
Well `None` doesn't say anything, this is very clear in what it will do so that's why i changed it